.github/ISSUE_TEMPLATE/learn-new-content-template.yaml

```
@@ -0,0 +1,63 @@
+name: Request updates or new content in Learn
```

```suggestion
name: Request new Apache Airflow content in Learn
```
"Updates" could sound like we're asking a user for bug fix info

---

Hmm yeah there is that danger when using the term "update". The reason for the 'updates' there is because we get a lot of issues that are "hey xyz is outdated/ there is a better way of doing this now/ xyz should be added to this list" etc. 

So neither a bug in the sense that something does not work but also not really new content, at least in my mind "new" means a new tutorial or at least a new section in a guide 🤔 

How about "Request new Apache Airflow content or content additions in Learn" ? 

cc: @kentdanas

---

Yeah....I agree most of the issues we get are "hey this part is outdated" or "hey can we add context on this additional thing here". However, the vast majority of these get raised to us in Slack and it's us that ends up creating the issue, so I wouldn't be too worried about people being confused here. Maybe we just keep it simple with Jake's suggestion, and can adjust later if needed.